plugins/woocommerce/src/Internal/EmailEditor/EmailApiController.php

```diff
@@ -24,9 +37,13 @@ public function get_email_data( $post_data ): array {
 		$email_type  = get_post_meta( $post_data['id'], Integration::WC_EMAIL_TYPE_ID_POST_META_KEY, true );
 		$post_option = get_option( "woocommerce_{$email_type}_settings" );
 
+		$email = $this->get_email_by_type( $email_type );
+
 		return array(
 			'subject' => $post_option['subject'] ?? null,
 			'heading' => $post_option['heading'] ?? null,
+			'default_subject' => $email->get_default_subject(),
+			'default_heading' => $email->get_default_heading(),
```

Comment:
Why do we need to pass these defaults as separate properties? Can't we just use them as a subject fallback directly?

```php
'subject' => $post_option['subject'] ?? $email->get_default_subject(),
```

Reply:
@costasovo The default values are used as placeholders, not as default values. Do you think we should use another name in the response here?

Reply:
@lysyjan Placeholders, I see. How does it work during sending, when a user doesn't save any Subject and just keeps the placeholder? What subject would be used?

Reply:
Sending uses the [WC_Email::get_subject()](https://github.com/woocommerce/woocommerce/blob/b9956f17234492c7272aed5f16684d0ca2062aa3/plugins/woocommerce/includes/emails/class-wc-email.php#L485-L505) method, which handles logic such as the use of the default subject if needed. I'd say the logic is the same as the old email setup page.